### PR TITLE
Handle null purchase lists in BillingWrapper#queryPurchases.

### PR DESCRIPTION
**Pull request by tonycosentini**
This is oddly nullable (as far as I can tell, it's nowhere in the documentation). This adds a null check, if i'ts null we'll just build the map off an empty list.

I added a unit test to cover this, but was not able to manually reproduce (maybe it depends on the state the device is in? or whatever version of Google Play Services the device running?)

### CODE REVIEW

**Comment by tonycosentini** [comment]
@vegaro @jeiting sorry for the double PR review request, accidentally hit the request button on the wrong PR

File: purchases/src/main/kotlin/com/revenuecat/purchases/BillingWrapper.kt [hunk 1]
```diff
@@ -253,9 +253,13 @@ internal class BillingWrapper internal constructor(
         return billingClient?.let {
             debugLog("[QueryPurchases] Querying $skuType")
             val result = it.queryPurchases(skuType)
+
+            // Purchases.PurchaseResult#purchasesList is not marked as nullable, but it does sometimes return null.
+            val purchasesList = if (result.purchasesList != null) result.purchasesList else emptyList<Purchase>()
```
Comment (vegaro) [inline purchases/src/main/kotlin/com/revenuecat/purchases/BillingWrapper.kt]: Can you use the Elvis operator here?

**Comment by jeiting** [comment]
For those following along at home the elvis operator is the ?:, presumably
since it looks like swoopy hair.
ᐧ

File: purchases/src/main/kotlin/com/revenuecat/purchases/BillingWrapper.kt (hunk 1 above)
Comment (tonycosentini) [reply purchases/src/main/kotlin/com/revenuecat/purchases/BillingWrapper.kt]: Done! (Also shows how much Kotlin I write 🤦‍♂ )

**Pull request merged**